SpatialGDK/Source/SpatialGDKFunctionalTests/Public/TestMaps/GeneratedTestMap.h

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "GenerateTestMapsCommandlet.h"
+#include "GeneratedTestMap.generated.h"
+
+/**
+ * The base class for automatically generating test maps.
+ * Descend from this and your class should automatically be picked up by the GenerateTestMapsCommandlet for generation.
+ */
+UCLASS()
```

Comment:
Should definitely add `Abstract` here since we have a non-default constructor that we are "forced" to use.